data_model/src/transaction.rs

```diff
@@ -342,34 +350,53 @@ impl Transaction {
         self.payload.nonce = Some(nonce);
         self
     }
+}
 
-    /// Sign transaction with the provided key pair.
-    ///
-    /// # Errors
-    ///
-    /// Fails if signature creation fails
-    #[cfg(feature = "std")]
-    pub fn sign(
-        mut self,
-        key_pair: iroha_crypto::KeyPair,
-    ) -> Result<Transaction, iroha_crypto::Error> {
-        let signature = SignatureOf::new(key_pair, &self.payload)?;
-        self.signatures.insert(signature);
-
-        Ok(Self {
-            payload: self.payload,
-            signatures: self.signatures,
-        })
-    }
+/// Structure represents transaction in non-trusted form.
```

Comment:
I think, this form is trusted

Reply:
```suggestion
/// Structure representing transactions in a non-trusted form.
```
or
```suggestion
/// Structure that represents transactions in a non-trusted form.
```
(or `in a trusted form` if Daniil is right)